lib/pub_sub/active_record/events.rb

```diff
@@ -4,6 +4,9 @@ module PubSub
   module ActiveRecord
     module Events
       class CallbackEvent < PubSub::BaseEvent
+        def initialize(payload)
+          super({ timestamp: Time.now(), **payload })
```

Comment:
dont need parens, feels kinda weird since we never do that in our other repos (here and elsewhere)

```suggestion
          super({ timestamp: Time.current, **payload })
```

Reply:
Sorry, been writing too much JS.